site/docs/components/tile.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 title: Tile
 navTitle: Tile
-summaryParagraph: A visually interesting item in a list consisting of a card, visual, title metadata, and call to action.
-tags: ["Card", "Grid item", "Featured item"]
+summaryParagraph: Tile guidelines describe the consistent visual treatment of content within a container. The layouts apply to various sizes of Tiles and cover content structure, spacing, hierarchy, positioning of action and interactions for content and data.
+tags: ["content layout", "card", "content card", "media card"]
```

Comment:
Does it matter if these are lowercase @ActuallyACat ?

Reply:
Good call out - it doesn't pose any technical issues but it is inconsistent with other documentation pages, e.g. https://cultureamp.design/components/checkbox-field/ I'm not sure if we have any guidance for language in Tags, [the closest thing I could find is this](https://cultureamp.design/language/check/):

> All titles and buttons should follow sentence case, i.e., should begin with a capital letter and only proper nouns are capitalized.

Reply:
I think given there's inconsistency in other parts of the app this should slide and we should do a separate tidy up of the tags.